lib/script-loader.php

```diff
@@ -77,8 +107,6 @@ function gutenberg_enqueue_global_styles_custom_css() {
 		wp_add_inline_style( 'global-styles', $custom_css );
 	}
 }
-remove_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles_custom_css' );
```

Comment:
Don't we still need to remove the `wp_enqueue_global_styles_custom_css` callback?

The plugin supports the latest two WordPress versions, so, assuming this is backported for 6.6, it's only when WordPress 6.7 is released that we can remove this (at that point, the plugin can assume the minimum version is 6.6, where that function is no longer relevant).

Reply:
Oooh good point, it'll still be in core until 6.6 so yeah, I guess it'll need removing.

Reply:
Actually checking the CSS output on the front end, I'd expect any custom CSS styles to be duplicated with this change (because `wp_enqueue_global_styles_custom_css` is still being enqueued in core) but they're not 🤔 I wonder why?